docs/UserGuide.adoc

```diff
@@ -361,8 +464,23 @@ Format: `theme THEME_NAME`
 ****
 * `THEME_NAME` must specify a valid theme, and is case-insensitive.
 * The current available themes are: `white`, `light`, and `dark`.
+* The default theme is `white`
 ****
 
+You can change the application to the white theme using `theme white`.
+
+image::ThemeWhite.png[width="700"]
+
+Bibliotek shows `Application theme changed to: white` to indicate that your theme change is successful.
+
+The same goes for `theme light`:
+
+image::ThemeLight.png[width="700"]
+
+And `theme dark`:
+
+image::ThemeDark.png[width="700"]
```

Comment:
Temporarily modify `DarkTheme.css` on line 402 to `.pills .label {`

Reply:
thats exactly what it says?

Reply:
Oops. *line 410 to `.pills-small .label {`